docs/background/project-del-not.md

```diff
@@ -0,0 +1,44 @@
+# Why you should not bother deleting projects
```

Comment:
I went for "bother" because I was thinking of a discussion with an imaginary (yet 100% real) user. In this discussion, right from the beginning, I want to pass a "forget about projects and enjoy life" message. But yeah, few people would actually want to have such a discussion with Yours Truly, so let's go with your title :)